spec/unit/game_spec.rb

```diff
@@ -0,0 +1,122 @@
+require './lib/game.rb'
+
+describe Game do 
+
+  let(:player){ double :player, 
+                name: 'Jake',
+                pins_rest: 5,
+                frame: 5,
+                roll: 1,
+                score: 100,
+                count_next: 1
+              }
+  let(:player_4_left){ double :player, 
+                name: 'Jake',
+                pins_rest: 4
+              }
+  let(:player_end){ double :player, 
+                name: 'Jake', 
+                pins_rest: 4,
+                frame: 10, 
+                roll: 2, 
+                score: 200, 
+                count_next: 0
+              }
+  let(:player_extra){ double :player,
+                count_next: 1
+              }
+
+  let(:subject){ described_class.new(player.name) }
+  let(:subject_4_left){ described_class.new(player_4_left.name, player_4_left.pins_rest) }
+  let(:subject_game_has_ended){ described_class.new(player_end.name, player_end.pins_rest, player_end.frame, player_end.roll, player_end.score, player_end.count_next) }
+  let(:subject_game_has_extra_roll){ described_class.new(player_end.name, player_end.pins_rest, player_end.frame, player_end.roll, player_end.score, player_extra.count_next) }
+
+  let(:list){ ['player', 'frame', 'roll', 'score', 'count_next'] }
+
+  it 'expects the instantiated class should hold these state instance variables' do
```

Comment:
Which state instance variables? Reading just this test description, it is not clear what to expect.